### PR TITLE
Fix "Sway Quickstart"

### PR DESCRIPTION
**Pull request by paulrberg**
Changelog:

- [x] Fix name of "Sway Quickstart" document
- [x] Remove all references to tests in "Sway Quickstart"
- [x] Use underscore in "fuel_toolchain.md" file name

I have removed all references to the tests because as of the latest version of `forc`, which v0.31.1 at the time of writing, the default project that gets created with `forc new project_name` doesn't contain a `tests` directory anymore.

### CODE REVIEW

**Review by mohammadfawaz — COMMENTED** [review]
This is great! Thank you for cleaning this up. Maybe we should point to the [testing with Rust section](https://fuellabs.github.io/sway/v0.31.1/testing/testing-with-rust.html) at the end?

**Comment by paulrberg** [comment]
Thanks @mohammadfawaz, glad you like it.

I pushed another commit to add a reference to the "Testing with Rust" section in the "Sway Quickstart" document.

File: docs/src/introduction/sway_quickstart.md [hunk 1]
```diff
@@ -196,46 +193,11 @@ Logs:
...
-     Running tests/harness.rs (target/debug/deps/integration_tests-7a2922c770587b45)
-
-running 1 test
-test can_get_contract_id ... ok
-
-test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.09s
-```
-
-Congratulations, you've just created and tested your first Sway smart contract 🎉.
+We will cover how to test your contract later but, if you are eager to take a look, see [Testing with Rust](../testing/testing-with-rust.md).
```
Comment (kayagokalp) [inline docs/src/introduction/sway_quickstart.md:200]: Maybe it would make sense to link both testing with rust and unit testing?

**Review by kayagokalp — COMMENTED** [review]
Hey @paulrberg, thanks for the clean-up!

File: docs/src/introduction/sway_quickstart.md (hunk 1 above)
Comment (paulrberg) [reply docs/src/introduction/sway_quickstart.md:200]: Hey @kayagokalp! Unit testing shows a 404 now.

Screenshot 2022-11-21 at 10 06 40 PM

Comment (mohammadfawaz) [reply docs/src/introduction/sway_quickstart.md:200]: Yes this was fixed on the `master` book so it should be okay now.
https://fuellabs.github.io/sway/master/testing/unit-testing.html

Comment (paulrberg) [reply docs/src/introduction/sway_quickstart.md:200]: Whoops. Added now, see the commit below.

**Review by mohammadfawaz — DISMISSED** [review]
Awesome, :shipit:

**Review by mohammadfawaz — APPROVED** [review]

**Review by kayagokalp — APPROVED** [review]
:shipit:

**Pull request merged**